docs/guides/granting-access.md

```diff
@@ -2,7 +2,7 @@
 
 ## Roles
 
-Infra allows granting different levels of access via **roles**, such as `view`, `edit` or `admin`. Each connector has different roles that can be used:
+Infra allows granting different levels of access via **roles**, such as `view`, `connector` or `admin`. Each connector has different roles that can be used:
```

Comment:
```suggestion
Infra allows granting different levels of access via the **roles** `view` or `admin`. Each connector has different roles that can be used:
```

Good catch on these docs, we don't need to document the connector role, it should be handled internally at the moment.